src/style.css

```
@@ -1 +1,90 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  }
+
+  @font-face {
+    font-family: Avenir;
+    font-weight: 300;
+    src: url(fonts/Avenir-Book.ttf);
+  }
+
+  @font-face {
+    font-family: Avenir;
+    font-weight: 900;
+    src: url(fonts/Avenir-Heavy.ttf);
+  }
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+border-color: #E1E7ED;
+color: #3D4E61;
+}
+
+.search::placeholder  {
+  font-weight: 300;
+  }
+
+ .search-big::placeholder {
+  font-size: 16px;
+  }
+
+  .search-small::placeholder {
+  font-size: 14px;
+  }
+
+ .search {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  margin-top: 20px;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 #3D4E611A;
+  position: relative;
+  font-family: Avenir, sans-serif;
+  }
```

Fix the indents everywhere
```suggestion
 .search {
    display: flex;
    align-items: center;
    width: 100%;
    margin-top: 20px;
    border-radius: 4px;
    border: 1px solid #E1E7ED;
    background: #fff;
    box-shadow: 0 1px 8px 0 #3D4E611A;
    position: relative;
    font-family: Avenir, sans-serif;
  }
```

---

I am fixed